fs/fs.go

```diff
@@ -349,9 +349,10 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 	go func() {
 		rErr := fmt.Errorf("failed to resolve target")
 		for _, s := range src {
-			sociDesc := ocispec.Descriptor{}
-			if desc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]; ok {
-				sociDesc = desc
+			sociDesc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]
+			if !ok {
+				rErr = errors.Wrapf(rErr, "failed to resolve layer %q from %q: %v", s.Target.Digest, s.Name, "no ztoc for the layer")
```

Comment:
We know some layers don't have corresponding zTOC because of the configuration below;

https://github.com/awslabs/soci-snapshotter/blob/a6759ef18fb5028daddbe704494eb76e7653fc9e/cmd/soci/commands/create.go#L53-L58

So is it worth to report?

Reply:
And in such case, we shouldn't report an error

Reply:
I reword the error/log message as suggested by @rdpsin below. I think it's worth to report.

We know some layers doesn't have zTOC layer, but suppose if there is a bug causing a large layer should have zTOC but it's not in `c.imageLayerToSociDesc`, we'll know it's due to this (ztoc missing) instead of `Resolve` failure.